source/blog/2022-06-16-getting-started-with-solidus-starter-frontend.html.markdown

```diff
@@ -0,0 +1,134 @@
+    <figure class="figure">
+      <img src="/blog/2022/06/16/getting-started-with-solidus-starter-frontend/10-home.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Home page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2022/06/16/getting-started-with-solidus-starter-frontend/20-product.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Product page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2022/06/16/getting-started-with-solidus-starter-frontend/30-cart.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Cart page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2022/06/16/getting-started-with-solidus-starter-frontend/40-login.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Login page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2022/06/16/getting-started-with-solidus-starter-frontend/50-checkout-address.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Address page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2022/06/16/getting-started-with-solidus-starter-frontend/60-checkout-delivery.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Delivery page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2022/06/16/getting-started-with-solidus-starter-frontend/70-checkout-payment.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Payment page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2022/06/16/getting-started-with-solidus-starter-frontend/80-checkout-confirm.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Confirm page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2022/06/16/getting-started-with-solidus-starter-frontend/90-checkout-complete.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Complete page</figcaption>
+    </figure>
+  </div>
+</div>
+
+In addition to generating views for authentication and storefront functionality,
+the application template will also generate
+
+* Mailers with their corresponding views
+* [View components](https://viewcomponent.org) for complex presentational logic
+* Unit and integrated specs for testing the frontend
+
+## What's next for Solidus Starter Frontend?
+
+For Solidus 4, SolidusStarterFrontend will become the recommended option for
+building the storefront of a new Solidus app. We're planning to remove
+SolidusFrontend from the solidus meta-package gem and move its code to a new
+repo. If your app depends on SolidusFrontend, you'll need to explicitly add
+`solidus_frontend` to your Gemfile in order to continue using it.
+
```

Comment:
Last thing while we wait for the cover: we could add that we are still working on a solution to make the extensions compatible with the new frontend.